DESCRIPTION

```diff
@@ -118,5 +118,5 @@ Collate:
     'zzz.R'
 Config/rextendr/version: 0.3.1
 VignetteBuilder: knitr
-Config/polars/LibVersion: 0.42.0
+Config/polars/LibVersion: 0.42.1
```

Comment:
Was this change necessary?
Isn't this flag unrelated to the Rust binary?

Reply:
Maybe it wasn't, I think I misinterpreted the failures in the first commit because I've been bitten before by compilation errors when I forgot to update the libversion